### PR TITLE
[fix bug 1201467] MPL 2.0 FAQ template not found

### PR DESCRIPTION
**Pull request by alexgibson**
- rename template directories for /MPL/ to lowercase for consistency

### CODE REVIEW

**Comment by alexgibson** [comment]
Strange, looks like git/github isn't picking up the case sensitive changes I made to the filenames / folder here :/

**Comment by alexgibson** [comment]
Ok that looks better, I had to remove the files and then re-add them again. Should be good now hopefully :+1:

File: bedrock/mozorg/urls.py [hunk 1]
```diff
@@ -186,18 +186,18 @@
     page('contribute/stories/michael', 'mozorg/contribute/story-michael.html'),
 
     page('MPL', 'mozorg/MPL/index.html'),
```
Comment (alexgibson) [inline bedrock/mozorg/urls.py]: Not to self - missed renaming this one

**Comment by alexgibson** [comment]
Thanks @malena - updated!

**Comment by malena** [comment]
r+

**Comment by alexgibson** [comment]
Pushed to demo2 to make sure URL's are all ok:

https://www-demo2.allizom.org/en-US/MPL/
https://www-demo2.allizom.org/en-US/MPL/historical/
https://www-demo2.allizom.org/en-US/MPL/licence-policy/
https://www-demo2.allizom.org/en-US/MPL/headers/
https://www-demo2.allizom.org/en-US/MPL/1.1/
https://www-demo2.allizom.org/en-US/MPL/1.1/FAQ/
https://www-demo2.allizom.org/en-US/MPL/1.1/annotated/
https://www-demo2.allizom.org/en-US/MPL/2.0/
https://www-demo2.allizom.org/en-US/MPL/2.0/FAQ/
https://www-demo2.allizom.org/en-US/MPL/2.0/Revision-FAQ/
https://www-demo2.allizom.org/en-US/MPL/2.0/combining-mpl-and-gpl/
https://www-demo2.allizom.org/en-US/MPL/2.0/differences/
https://www-demo2.allizom.org/en-US/MPL/2.0/permissive-code-into-mpl/

All seem to be loading fine - @malena care to merge? Thanks!

**Pull request merged**